fsspec/implementations/dbfs.py

```diff
@@ -264,9 +264,9 @@ def _send_to_api(self, method, endpoint, json):
             try:
                 exception_json = e.response.json()
             except Exception:
-                raise e
+                raise e from None
```

Comment:
This is weird, no?
Is this to suppress the "while an exception was being handled" message